go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/nydus-snapshotter
 
-go 1.17
+go 1.18
```

Comment:
Why upgrade to `1.18`?

Reply:
sorry, my go environment is used by default